profile.md

```diff
@@ -441,16 +417,19 @@ We aim to define a common set of authorizations (particularly storage-related au
 
 For a given storage resource, the defined authorizations include:
 
-
-
-*   **storage.read**: Read data.  Only applies to “online” resources such as disk (as opposed to “nearline” such as tape where the **stage** authorization should be used in addition).
+*   **storage.read**: Read data. For “nearline” data such as tape, the **stage** authorization should be used in addition.
 *   **storage.create**: Upload data.  This includes renaming files if the destination file does not already exist. This capability includes the creation of directories and subdirectories at the specified path, and the creation of any non-existent directories required to create the path itself (note the server implementation MUST NOT automatically create directories for a client). This authorization DOES NOT permit overwriting or deletion of stored data.  The driving use case for a separate `storage.create` scope is to enable stage-out of data from jobs on a worker node.
 *   **storage.modify**: Change data.  This includes renaming files, creating new files, and writing data.  This permission includes overwriting or replacing stored data in addition to deleting or truncating data.  This is a strict superset of `storage.create`.
-*   **storage.stage**: Read the data, potentially causing data to be staged from a nearline resource to an online resource. This is a superset of `storage.read`.
-
-For a given computing resource, the defined authorization activities include:
+*   **storage.stage**: Stage the data from a nearline resource to an online resource, making the data available for access
+    to users bearing a token with the **storage.read** authorization. Storage Endpoints may impose additional constraints
+    on who is authorized to stage data.
 
+For nearline data such as tape, the “stage+transfer” semantics are defined by the tape access protocol independently
+from the authorization scheme. For XRootD, this means “prepare -s” (stage), “query prepare”, “abort prepare”, “evict”
+and “copy”. For HTTP, the semantics are defined in the
+[WLCG Tape REST API](https://cernbox.cern.ch/pdf-viewer/public/vLhBpHDdaXJSqwW/WLCG%20Tape%20REST%20API%20reference%20document.pdf).
```

Comment:
Suggestion to use the [wlcg-storage/wlcg-tape-rest-api](https://github.com/wlcg-storage/wlcg-tape-rest-api) repository instead of CERNBox for the WLCG Tape REST API specification document.